thrift/doc/mstch.md

```diff
@@ -132,7 +132,7 @@ about the context.
 
 Lists represent zero or more elements of some other type, and function similar
 to for-each loops in other languages. The body of the loop will be expanded
-once per element in the list, and on each iteration the context will be
+once per element in the list, and on each iteration, the context will be
```

Comment:
Existing is better; revert.

Alternatively,
```suggestion
once per element in the list, and, on each iteration, the context will be
```

Reply:
Added alternative